### DISCUSSION TITLE
Questions about the "download archive" option

### DISCUSSION THREAD

**Discussion by Miala-python** [JunkFood02/Seal#1609] [Q&A]
Hello,
When I explored the different options, I see this one:

**Download archive**
Save videos ID in an archive to avoid duplicates. (Translate from french to english, not sure correct 😅)

![Screenshot_2024-08-03-08-29-43-012_com.junkfood.seal.png](https://github.com/user-attachments/assets/3facd83f-f8a4-454f-aab8-52002b92f5fb)

What exactly does this option and where can we access this archive ?

I think it can be helpful when Seal skip some videos because of low connection.

Thank you !

**Comment by JunkFood02** [comment]
It's in the internal directory of the app and could not be accessed directly by users, but you can just click on this option to see which videos have been recorded in this file
